src/Skeleton/Skeleton.stories.tsx

```diff
@@ -0,0 +1,206 @@
+// REACT
+import * as React from 'react';
+// STORYBOOK
+import { storiesOf } from '@storybook/react';
+import { boolean, select, text } from '@storybook/addon-knobs';
+// VENDOR
+import styled, { ThemeProvider } from '@xstyled/styled-components';
+import { th } from '@xstyled/system';
+// ANCHOR
+import * as README from './README.md';
+import { Typography, Avatar, Button, Close } from '..';
+import { RootTheme } from '../theme';
+import { typography, TypographyTags, Scale } from '../theme/typography.theme';
+import { Skeleton } from './Skeleton.component';
+
+const StyledStory = styled('div')`
+    padding: 2rem 4rem;
+    color: text.body;
+    font-family: ${th('typography.fontFamily')};
+`;
+
+const Card = styled('div')`
+    width: 330px;
+    background-color: white;
+    border-radius: base;
+    box-shadow: 0px 1px 4px 0px rgba(0, 0, 0, 0.2);
+    display: flex;
+    flex-direction: column;
+`;
+
+const CardHeader = styled('div')`
+    display: flex;
+    flex-direction: row;
+    align-items: center;
+    padding: 1rem;
+    position: relative;
+
+    button {
+        position: absolute;
+        top: 0.5rem;
+        right: 0.5rem;
+    }
+
+    .anchor-avatar {
+        background-color: #ec5673;
+    }
+`;
+
+const CardInfo = styled('div')`
+    margin-left: 1rem;
+    display: flex;
+    flex-grow: 1;
+    flex-direction: column;
+    justify-content: center;
+`;
+
+const CardImage = styled('div')`
+    width: 100%;
+    height: 190px;
+    background: linear-gradient(170deg, #df206c 0%, #f16667 100%);
+`;
+const CardContent = styled('div')`
+    width: 100%;
+`;
+const CardFooter = styled('div')`
+    padding: 1rem 1rem 1.5rem;
+`;
+
+type TagSelect = TypographyTags | '';
+type ScaleSelect = Scale | '';
+const Tags = [''].concat(Object.keys(typography.tag)) as TagSelect[];
+const Scales = ['']
+    .concat(Object.keys(typography.scale))
+    .map(x => parseInt(x, 10) || '') as ScaleSelect[];
+
+storiesOf('Components/Skeleton', module)
+    .addParameters({
+        readme: {
+            sidebar: README,
+        },
+    })
+    .add('Text', () => {
+        const loading = boolean('loading', true);
+        const defaultText =
+            // tslint:disable-next-line: max-line-length
+            'For a long time in software companies the practice of building software as one-off initiatives at has been called into question and deemed insular. Its no longer acceptable to deliver a piece of software that solves a particular problem and call it a solution. As new requirements emerge, narrowing our focus just to a particular challenges requirements results in organizational tension, substandard solutions, churn, code rot, technical debt, etc.';
```

Comment:
You can get around the line length by using `` rather than '' for strings. Can wrap to multiple lines.

Reply:
I had that originally, but realized that it was causing the skeleton text to be way too long (because of all the added indentation spaces).

Instead of avoiding the issue, I should probably just fix it though...